solcjs

```diff
@@ -15,17 +15,39 @@ process.removeAllListeners('uncaughtException');
 var commander = require('commander');
 
 const program = new commander.Command();
+const parseOptimizeRuns = function(value) {
+  const parsedValue = parseInt(value, 10);
+  if (isNaN(parsedValue)) {
+    throw new commander.InvalidArgumentError('The "runs" setting must be an unsigned number');
```

Comment:
I wonder if this function should be just a generic `commanderParseInt` function?

Reply:
Yeah. That's a good idea. This could be a generic function for validating unsigned ints. The message here does not even have to include the name of the argument or value because commander will prepend that to the message it displays.

Reply:
Updated